src/workerinfo.js

```diff
@@ -34,57 +35,65 @@ class WorkerInfo {
     }
   }
 
-  async seen(provisionerId, workerType) {
+  async seen(provisionerId, workerType, workerGroup, workerId) {
     const expired = entity => Date.now() - new Date(entity.expires) > 24 * 60 * 60 * 1000;
     const expires = taskcluster.fromNow('5 days');  // temporary hard coded expiration
```

Comment:
@djmitche Agreed, we are currently waiting 24 hours after an entry is expired to write the updated value. I am assuming we want to update the expiration value if there's less than 24 hours left to expiration. Also, is 3 days for the expiration time short enough for workers?

```js
const expires = workerId ? taskcluster.fromNow('3 days') : taskcluster.fromNow('5 days');
const expired = entity => new Date(entity.expires) - Date.now() < 24 * 60 * 60 * 1000;
```